docs/typescript.md

```diff
@@ -184,3 +225,30 @@ const Button = styled('button')`
 
 export default Button
 ```
+
+## preact-emotion
+
+Types are almost same with `react-emotion` package.
```

Comment:
If they're almost the same and not the exact same, could you explain how they're different?

Reply:
OK. I will add following.
> However, this typing use Preact component types like `ComponentConstructor` and `FunctionalComponent` instead of React component types.

Is this enough?